src/stories/List.stories.tsx

```diff
@@ -15,9 +19,14 @@ export default ListMeta;
 
 export const PrimaryList = () => (
   <List>
```

Comment:
What do you think about holding onto a selected state here that could identify the selected list item? This would allow us to test the click functionality.
```
const [selectedListItem, setSelectedListItem] = useState(2);
```

Reply:
Thats a great idea